htsget_server/beacon_openapi.yaml

```diff
@@ -864,7 +864,7 @@ components:
                 description: Reference to the bioinformatics analysis ID (`analysis.id`)
                 type: string
               biosampleId:
-                description: Reference to biosample ID (`biosample.id`)
+                description: Reference to biosample ID (`biosample.id`). For MoH, this will be `{program_id}~{submitter_sample_id}`, delimited with a tilde. If this is not available, it will be the name of the sample as listed in the variant file.
```

Comment:
are we 100% confident that submitter_sample_ids will be unique across programs? or do we also need donor_ids to ensure this?

Reply:
That's what it says in the spreadsheet...